framework/egl/wrapper/eglwEnums.inl

```diff
@@ -238,6 +238,7 @@
 #define EGL_LOSE_CONTEXT_ON_RESET_EXT						0x31BF
 #define EGL_PLATFORM_X11_EXT								0x31D5
 #define EGL_PLATFORM_X11_SCREEN_EXT							0x31D6
+#define EGL_PLATFORM_WAYLAND_KHR							0x31D8
```

Comment:
thank you for kindly explain. I have updated scripts/egl/common.py and I remained framework/egl/wrapper/eglwEnums.inl file which is generated by scripts/gen_egl.py . if you want to remove it, I'll remove EGL_PLATFORM_WAYLAND_KHR line form eglwEnums.inl